src/khoj/database/models/__init__.py

```diff
@@ -202,11 +202,16 @@ class ModelType(models.TextChoices):
         ANTHROPIC = "anthropic"
         GOOGLE = "google"
 
+    class PriceTier(models.TextChoices):
```

Comment:
We can define the `PriceTier` class outside of the data model class so that it can be reused across data models.